### PR TITLE
[sql] Add Gigas Bracelets Desynth

### PR DESCRIPTION
**Pull request by 9001-Sols**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

Updated sql script to add desynthesis recipe for `Gigas Bracelets`

## Sources

- https://wiki.ffo.jp/html/7184.html
- https://ffxiclopedia.fandom.com/wiki/Gigas_Bracelets?oldid=319936

## Proof of level 57 Goldsmithing
I started a new character and leveled Goldsmithing 1 level at a time, attempting the synth with +3 Synth Support to find exactly what level it became synthable. Here are the last 2 videos showing it was not synthable at 29, but became synthable at 30, giving us the true level of the synth (57).

image

https://github.com/user-attachments/assets/72d7c3c9-2bda-4767-a2ba-dca94f369df0

https://github.com/user-attachments/assets/b49278af-0c3a-4c5c-a65f-f63ca685bc2e

## Steps to test these changes

- Be on a goldsmither 57+
- Use a lightning crystal on `Gigas Bracelets`

## Screenshots

image

image

### CODE REVIEW

**Comment by Xaver-DaRed** [comment]
Changes look good, ID wise.

Click the checkboxes accepting LSB's conditions and please, in the future, do not use a different repository (ASB) PR schema.

**Comment by Xaver-DaRed** [comment]
So, after checking the links and digging around I can find absolutely no evidence of the recipe level.

So I cannot approve this in good faith. Most we can accept is setting the recipe's goldsmithing level to 255, effectively existing in a disabled state, with a comment stating the recipe level needs to be retail verified.

File: sql/synth_recipes.sql [hunk 1]
```diff
@@ -4755,6 +4755,7 @@ INSERT INTO `synth_recipes` VALUES (75530,0,0,0,0,0,0,0,0,10,0,4099,4241,2773,27
 INSERT INTO `synth_recipes` VALUES (75531,0,0,0,0,0,0,0,0,10,0,4099,4241,2773,2773,2774,2774,2775,4161,0,0,2781,2781,2781,2781,1,1,1,1,'Enfeeb. Kit Sleep');
 INSERT INTO `synth_recipes` VALUES (75532,1,0,0,0,0,0,34,0,0,0,4100,4242,12994,0,0,0,0,0,0,0,818,818,848,848,2,3,1,2,'Shoes (desynth)');
 INSERT INTO `synth_recipes` VALUES (75533,1,0,0,0,0,25,0,0,0,0,4100,4242,12722,0,0,0,0,0,0,0,834,819,819,819,2,5,6,7,'Bracers (desynth)'); -- FFXIclopedia
+INSERT INTO `synth_recipes` VALUES (75534,1,0,0,0,60,0,0,0,0,0,4100,4242,14018,0,0,0,0,0,0,0,653,653,745,745,1,2,1,1,'Gigas Bracelets (desynth)');
```
Comment (Xaver-DaRed) [inline sql/synth_recipes.sql]: Please set the goldsmithing level to 255 and add a comment stating the recipe needs retail verification for its lvl.

**Review by Xaver-DaRed — CHANGES_REQUESTED** [review]

**Comment by Xaver-DaRed** [comment]
An additional note: We cannot approve untested PRs. (Since the checkbox is unchecked)

**Comment by 9001-Sols** [comment]
> An additional note: We cannot approve untested PRs. (Since the checkbox is unchecked)

I haven't had a chance to spin up LSB yet, all my development was done in ASB and I was asked to port the PRs over to here. As soon as I get LSB spun up and I have time to test, I'll check the box.

File: sql/synth_recipes.sql (hunk 1 above)
Comment (9001-Sols) [reply sql/synth_recipes.sql]: I captured data from retail and will update my summary with the details. The correct level was 57.

**Review by Xaver-DaRed — APPROVED** [review]
I tested this myself after seing the evidence.

However, this is a one time thing. Next time you will need to test this yourself or find some1 willing to test it.

**Comment by 9001-Sols** [comment]
> I tested this myself after seing the evidence.
> 
> However, this is a one time thing. Next time you will need to test this yourself or find some1 willing to test it.

Hey sorry, not trying to dump testing efforts on others.

Right now LSB is hosed when I launch it locally, as the retail install to capture data messed up LSB/ASB/Horizon setup. I log into a debug zone and GM commands don't work. I'm going to remake the database and try to do a fresh setup. I just haven't had time. As soon as I have LSB functioning I'll retest all my PRs (they were originally tested on ASB) and go ahead and check the box that things were tested.

**Pull request merged**